kumascript/macros/DeprecatedGeneric.ejs

```diff
@@ -20,16 +20,16 @@ if ($1) {
     var t = $1.toLowerCase();
     var n = "";
 
-    if ( string.startswith(t, "js") ) {
-        n = string.trim( string.substr(t, 2) );
+    if (string.startswith(t, "js")) {
+        n = string.trim( string.substr(t, 2));
         ver = "JavaScript ";
         if (string.length(n) > 0) {
-            if(wiki.pageExists(lang + '/docs/JavaScript/New_in_JavaScript/' + n)) {
+            if(true) {
```

Comment:
code smell is still there. All of these `if` statements should just become
```
link = '/' + lang + '/docs/JavaScript/New_in_JavaScript/' + n;
```

Reply:
Ah, yes. Looks like my `git grep -n "if (true) {" missed a few spots that still needed fixing. Will update this shortly